models/business-unit/src/associate/types.ts

```diff
@@ -0,0 +1,24 @@
+import type {
+  Associate,
+  AssociateDraft,
+  Customer,
+} from '@commercetools/platform-sdk';
+import { TReferenceGraphql } from '@commercetools-test-data/commons';
+import type { TBuilder } from '@commercetools-test-data/core';
+
+export type TAssociateDraft = AssociateDraft;
+export type TAssociateDraftGraphql = TAssociateDraft;
+
+export type TAssociateDefault = Omit<Associate, 'customer'> & {
+  customer: Customer;
+};
+export type TAssociateRest = Associate;
+export type TAssociateGraphql = Omit<TAssociateDefault, 'customer'> & {
```

Comment:
I don't think we want to omit `customer`
![image](https://github.com/commercetools/test-data/assets/8136478/b6344c4d-2497-4bfb-bb02-3615bcc6a463)